be/src/exec/pipeline/aggregate/sorted_aggregate_streaming_sink_operator.cpp

```diff
@@ -0,0 +1,59 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+#include "exec/pipeline/aggregate/sorted_aggregate_streaming_sink_operator.h"
+
+#include "exec/vectorized/sorted_streaming_aggregator.h"
+#include "runtime/current_thread.h"
+
+namespace starrocks::pipeline {
+SortedAggregateStreamingSinkOperator::SortedAggregateStreamingSinkOperator(
+        OperatorFactory* factory, int32_t id, int32_t plan_node_id, int32_t driver_sequence,
+        std::shared_ptr<SortedStreamingAggregator> aggregator)
+        : Operator(factory, id, "sorted_aggregate_streaming_sink", plan_node_id, driver_sequence),
+          _aggregator(std::move(aggregator)) {
+    _aggregator->ref();
+}
+
+Status SortedAggregateStreamingSinkOperator::prepare(RuntimeState* state) {
+    RETURN_IF_ERROR(Operator::prepare(state));
+    RETURN_IF_ERROR(_aggregator->prepare(state, state->obj_pool(), _unique_metrics.get(), _mem_tracker.get()));
+    return _aggregator->open(state);
+}
+
+void SortedAggregateStreamingSinkOperator::close(RuntimeState* state) {
+    _aggregator->unref(state);
+    Operator::close(state);
```

Comment:
need call `_aggregator->close()`?

Reply:
our selection use selection[i] == 0 as selected. I keep our interfaces consistent.
ref:

    void update_batch_selectively(FunctionContext* ctx, size_t chunk_size, size_t state_offset, const Column** columns,
                                  AggDataPtr* states, const std::vector<uint8_t>& filter) const override {
        for (size_t i = 0; i < chunk_size; i++) {
            // TODO: optimize with simd ?
            if (filter[i] == 0) {
                static_cast<const Derived*>(this)->update(ctx, columns, states[i] + state_offset, i);
            }
        }
    }

Reply:
Aggregator::close will be called in _aggregator->unref

Reply:
```
class ContextWithDependency {
......
    void unref(RuntimeState* state) {
        if (_num_running_operators.fetch_sub(1, std::memory_order_acq_rel) == 1) {
            close(state);
        }
    }
```